src/index.js

```diff
@@ -0,0 +1,49 @@
+import pluralize, { singular } from 'pluralize';
+
+function normalize(slug: string): string {
```

Comment:
Curious - does flow play nicely with composed functions, given its argument isn't defined at declaration?

Reply:
I don't really understand the question

Reply:
https://github.com/mb3online/expresso/blob/8a2ca65e8bee896fe040c18dc67e1486cd1f6e94/services/model/reduce-relationships.js#L5